solara/website/assets/custom.css

```
@@ -239,6 +239,16 @@ blockquote p:last-child {
     border-radius: 8px;
 }
 
+.v-navigation-drawer__content .algolia{
```

```suggestion
.v-navigation-drawer__content .algolia {
```
guess we do not have prettier yet!